tests/utils.go

```diff
@@ -0,0 +1,421 @@
+package tests
```

Comment:
How did we end up with this utils file?

Either the github diff is "broken" for this case or I can not find the source. But I can see a lot of new functions being added. 

Is it migrated from some other tests/utils?

Reply:
Ok, I Get it. The diff here is shown to the current main. A new PR was just merged
https://github.com/kubevirt/containerized-data-importer/pull/2310

@alromeros I wonder if an automatic merge will work ok in this case. Or maybe you need to rebase to make sure it is correct!

Reply:
Ok I have not seen your previous comment, you replied so fast ;)

Reply:
the rest looks good